Update develop-storage-files-storage-access-control.md

Max-Meng opened the pull request:
-Update a few occurrences of user_name to sql_login_name or  sql_user_name to avoid mis-understanding. 
-Add suffix 'in the master database' and 'in the user database' accordingly to avoid confusion.

Max-Meng committed:
Update develop-storage-files-storage-access-control.md

-Update a few occurrences of user_name to sql_login_name or 
sql_user_name to avoid mis-understanding. 
-Add suffix 'in the master database' and 'in the user database' accordingly to avoid confusion.

filippopovic reviewed (changes requested):
In SQL, there are SQL logins and AAD logins (logins tied to AAD account). These statements are valid for both AAD and SQL logins. @Max-Meng

Max-Meng committed:
Update develop-storage-files-storage-access-control.md

Update my change request from sql_login_name to login_name, and   sql_user_name to user_name.

Max-Meng committed:
Update develop-storage-files-storage-access-control.md

remove a few 'sql' prefix to make the content also applies to AAD logins.

Max-Meng committed:
Merge branch 'MicrosoftDocs:main' into patch-10
